Geometry/VeryForwardGeometryBuilder/python/GeometryTest_cfg.py

```diff
@@ -6,7 +6,7 @@
 
 # geometry
 process.load("Geometry.VeryForwardGeometry.geometryRP_cfi")
-process.XMLIdealGeometryESSource.geomXMLFiles.append('Geometry/VeryForwardData/data/RP_Garage/RP_Dist_Beam_Cent.xml')
```

Comment:
not for this PR, but worth noticing:
full cfg.py files do not belong in /python directories. They should be in /test or something similar.